lib/condorSecurity.py

```diff
@@ -189,12 +255,23 @@ def enforce_requests(self):
 
 ########################################
 class EnvProtoState(SecEnvState):
```

Comment:
Should the inherited class be mentioned in the docstring?

Reply:
If you mean the parent, it's in the signature